modules/apps/segments/segments-content-targeting-upgrade/src/main/java/com/liferay/segments/content/targeting/upgrade/internal/upgrade/v1_0_0/ContentTargetingUpgradeProcess.java

```diff
@@ -36,11 +36,13 @@
 public class ContentTargetingUpgradeProcess extends UpgradeProcess {
 
 	public ContentTargetingUpgradeProcess(
-		RuleConverterRegistry ruleConverterRegistry,
-		SegmentsEntryLocalService segmentsEntryLocalService) {
+		SegmentsEntryLocalService segmentsEntryLocalService,
+		SegmentsContentTargetingUpgradeStepRegistrator
```

Comment:
Hey @GabrielHenriquedev, pass the `ServiceTracker` reference instead of the `SegmentsContentTargetingUpgradeStepRegistrator` one.